src/main/java/ro/societateahermes/backendservice/controller/controllerImplementation/CDMemberControllerImplementation.java

```diff
@@ -0,0 +1,65 @@
+package ro.societateahermes.backendservice.controller.controllerImplementation;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.FieldError;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.*;
+
+import ro.societateahermes.backendservice.controller.CDMemberControllerInterface;
+import ro.societateahermes.backendservice.entities.CDMember;
+import ro.societateahermes.backendservice.entities.DTO.CDMemberDTO;
+import ro.societateahermes.backendservice.service.serviceImplementation.CDMemberServiceImplementation;
+
+import javax.validation.Valid;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@RestController
+@RequestMapping("/cdMember")
+public class CDMemberControllerImplementation implements CDMemberControllerInterface {
+    private final CDMemberServiceImplementation CDMemberService;
+
+    public CDMemberControllerImplementation(CDMemberServiceImplementation serviceImplementation) {
+        CDMemberService = serviceImplementation;
+    }
+
+    @PostMapping("/CD")
+    @ResponseStatus(HttpStatus.CREATED)
+    public CDMember saveCDMember(@Valid @RequestBody CDMember CDMember) {
+        return CDMemberService.save(CDMember);
+    }
+
+    @GetMapping
+    @Override
+    public List<CDMember> getAllCDMembers() {
+        return CDMemberService.getAllCDMembers();
+    }
+
+    @DeleteMapping("/{CDMemberID}")
+    @Override
+    public void deleteCDMember(@PathVariable("CDMemberID") Long CDMemberID) {
+        CDMemberService.delete(CDMemberID);
+    }
+
+    @PutMapping("/{cdID}")
+    @ResponseStatus(HttpStatus.CREATED)
```

Comment:
ca mai sus cu url-ul

Reply:
@McShenanigans  same

Also, @mircea216 statusul nu ar trebui sa fie CREATED (204), din cauza ca tu nu ai adaugat nimic nou in db, doar ai schimbat, ceea ce s-ar traduce intr-un OK (200)

Reply:
am modificat cu OK instead of CREATED